### PR TITLE
HeatmapNG: cell value filtering and color clamping

### PR DESCRIPTION
**Pull request by leeoniya**
Supports filtering:
image

And color range:
image

### CODE REVIEW

File: public/app/plugins/panel/heatmap-new/HeatmapPanel.tsx [hunk 1]
```diff
@@ -126,7 +126,10 @@ export const HeatmapPanel: React.FC<HeatmapPanelProps> = ({
       getTimeRange: () => timeRangeRef.current,
       palette,
       cellGap: options.cellGap,
-      hideThreshold: options.filterValues?.min, // eventually a better range
+      hideLE: options.filterValues?.min,
+      hideGE: options.filterValues?.max,
```
Comment (leeoniya) [inline public/app/plugins/panel/heatmap-new/HeatmapPanel.tsx:130]: @ryantxu does `filterValues` min/max seem backwards to you?

maybe `hideValues.le` and `hideValues.ge`?

or `showValues`? though we might use this one for actually rendering text in the cells in the future :roll_eyes:

Comment (ryantxu) [reply public/app/plugins/panel/heatmap-new/HeatmapPanel.tsx:130]: 🤔 maybe:

```
hideCells: {
  le?: number;
  ge?: number;
},
```
buckets, bins, and cells 😱 

"Values" is the value inside

File: public/app/features/transformers/calculateHeatmap/editor/AxisEditor.tsx [hunk 2]
```diff
@@ -24,11 +24,6 @@ const logModeOptions: Array<SelectableValue<HeatmapCalculationMode>> = [
     value: HeatmapCalculationMode.Size,
     description: 'Split the buckets based on size',
   },
-  {
-    label: 'Count',
```
Comment (ryantxu) [inline public/app/features/transformers/calculateHeatmap/editor/AxisEditor.tsx:28]: This does not do anything different than split, so lets hide it

File: public/app/plugins/panel/heatmap-new/migrations.ts [hunk 3]
```diff
@@ -69,8 +74,8 @@ export function angularToReactHeatmap(angular: any): { fieldConfig: FieldConfigS
       ...defaultPanelOptions.color,
       steps: 128, // best match with existing colors
     },
-    cellGap: asNumber(angular.cards?.cardPadding),
-    cellSize: asNumber(angular.cards?.cardRound),
+    cellGap: asNumber(angular.cards?.cardPadding, 2),
+    cardRadius: asNumber(angular.cards?.cardRound), // just to keep it
```
Comment (leeoniya) [inline public/app/plugins/panel/heatmap-new/migrations.ts:78]: cellRadius?

**Review by ryantxu — APPROVED** [review]
getting there

**Pull request merged**